src/entities/navigation/navigation.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from "./navigation.module.css";
+import { categoriesApi } from "~entities/categories";
+import { ICategory } from "~shared/api/models/types.ts";
+import { categoriesKeys } from "~entities/categories/api/categoriesApi.ts";
+import { Link } from "react-router-dom";
+import { PAGE_PATH } from "~shared/config";
+import { useQueryClient } from "@tanstack/react-query";
+import cn from "classnames";
+
+interface IProps {
+  orientation?: 'vertical' | undefined
+}
+
+export function Navigation(props: IProps) {
+  const queryClient = useQueryClient();
+  categoriesApi.useCategories();
+  const categories = queryClient.getQueryData<ICategory[]>(categoriesKeys.categories.getCategories());
```

Comment:
It’s better to hardcode the list of categories in the top menu. The number of these categories can be huge and they cannot all fit to the list. If you want to control these categories on the backend, then it is better to add a new “public” field that will control the visibility of the category in the menu

Reply:
Yeah. I think about add fields like 'order', 'showInMenu' and return with request only 3 or 4 categories